sessionctx/variable/sysvar.go

```diff
@@ -125,6 +125,10 @@ var defaultSysVars = []*SysVar{
 		s.enforceMPPExecution = TiDBOptOn(val)
 		return nil
 	}},
+	{Scope: ScopeSession, Name: TiFlashMaxThreads, Type: TypeInt, Value: strconv.Itoa(DefTiFlashMaxThreads), MinValue: -1, MaxValue: MaxConfigurableConcurrency, SetSession: func(s *SessionVars, val string) error {
+		s.TiflashMaxThreads = TidbOptInt64(val, DefTiFlashMaxThreads)
+		return nil
+	}},
```

Comment:
Just this change, and then LGTM :-)
```suggestion
	{Scope: ScopeGlobal | ScopeSession, Name: TiFlashMaxThreads, Type: TypeInt, Value: strconv.Itoa(DefTiFlashMaxThreads), MinValue: -1, MaxValue: MaxConfigurableConcurrency, SetSession: func(s *SessionVars, val string) error {
		s.TiflashMaxThreads = TidbOptInt64(val, DefTiFlashMaxThreads)
		return nil
	}},
```

(Please also move it in the file to where global | session variables are)

Reply:
done.  :)